From ad4d22cd6b0b1237a4d50699736d38cd3c902ace Mon Sep 17 00:00:00 2001
From: Xiangyu Chen <xiangyu.chen@windriver.com>
Date: Wed, 12 Mar 2025 00:16:24 -0700
Subject: [PATCH] Makefile.am: do not compile the code which was generated from
 test-full.proto in protobuf-c-native

Those code was auto generated by protoc command with test-full.proto, those code are not compatible
with protobuf 4.22.x, so temporarily disable compile those code until protobuf-c upstream adapt
the test-full.proto with latest version protobuf.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Xiangyu Chen <xiangyu.chen@windriver.com>

Rebase to 1.5.1
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 Makefile.am | 75 -----------------------------------------------------
 1 file changed, 75 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 77aa9d9..5cdd81a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -138,81 +138,6 @@ else
 
 LOG_COMPILER = $(VALGRIND)
 
-check_PROGRAMS += \
-	t/generated-code/test-generated-code \
-	t/generated-code2/test-generated-code2 \
-	t/generated-code3/test-generated-code3 \
-	t/version/version
-
-TESTS += \
-	t/generated-code/test-generated-code \
-	t/generated-code2/test-generated-code2 \
-	t/generated-code3/test-generated-code3 \
-	t/version/version
-
-t_generated_code_test_generated_code_SOURCES = \
-	t/generated-code/test-generated-code.c \
-	t/test.pb-c.c
-t_generated_code_test_generated_code_LDADD = \
-	protobuf-c/libprotobuf-c.la
-
-t_generated_code2_test_generated_code2_SOURCES = \
-	t/generated-code2/test-generated-code2.c \
-	t/test-full.pb-c.c \
-	t/test-optimized.pb-c.c
-t_generated_code2_test_generated_code2_LDADD = \
-	protobuf-c/libprotobuf-c.la
-
-t_generated_code3_test_generated_code3_CPPFLAGS = \
-	-DPROTO3
-
-t_generated_code3_test_generated_code3_SOURCES = \
-	t/generated-code/test-generated-code.c \
-	t/test-proto3.pb-c.c
-
-t_generated_code3_test_generated_code3_LDADD = \
-	protobuf-c/libprotobuf-c.la
-
-noinst_PROGRAMS += \
-	t/generated-code2/cxx-generate-packed-data
-
-t_generated_code2_cxx_generate_packed_data_SOURCES = \
-	t/generated-code2/cxx-generate-packed-data.cc \
-	t/test-full.pb.cc \
-	protobuf-c/protobuf-c.pb.cc
-$(t_generated_code2_cxx_generate_packed_data_OBJECTS): t/test-full.pb.h
-t_generated_code2_cxx_generate_packed_data_CXXFLAGS = \
-	$(AM_CXXFLAGS) \
-	$(protobuf_CFLAGS)
-t_generated_code2_cxx_generate_packed_data_LDADD = \
-	$(protobuf_LIBS)
-
-t/test.pb-c.c t/test.pb-c.h: $(top_builddir)/protoc-gen-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test.proto
-	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-gen-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test.proto
-
-t/test-optimized.pb-c.c t/test-optimized.pb-c.h: $(top_builddir)/protoc-gen-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test-optimized.proto
-	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-gen-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test-optimized.proto
-
-t/test-full.pb-c.c t/test-full.pb-c.h: $(top_builddir)/protoc-gen-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test-full.proto
-	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-gen-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test-full.proto
-
-t/test-full.pb.cc t/test-full.pb.h: @PROTOC@ $(top_srcdir)/t/test-full.proto
-	$(AM_V_GEN)@PROTOC@ -I$(top_srcdir) --cpp_out=$(top_builddir) $(top_srcdir)/t/test-full.proto
-
-t/test-proto3.pb-c.c t/test-proto3.pb-c.h: $(top_builddir)/protoc-gen-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test-proto3.proto
-	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-gen-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test-proto3.proto
-
-t/generated-code2/test-full-cxx-output.inc: t/generated-code2/cxx-generate-packed-data$(EXEEXT)
-	$(AM_V_GEN)$(top_builddir)/t/generated-code2/cxx-generate-packed-data$(EXEEXT) > $(top_builddir)/t/generated-code2/test-full-cxx-output.inc
-
-BUILT_SOURCES += \
-	t/test.pb-c.c t/test.pb-c.h \
-	t/test-full.pb-c.c t/test-full.pb-c.h \
-	t/test-optimized.pb-c.c t/test-optimized.pb-c.h \
-	t/test-full.pb.cc t/test-full.pb.h \
-	t/test-proto3.pb-c.c t/test-proto3.pb-c.h \
-	t/generated-code2/test-full-cxx-output.inc
-
 t_version_version_SOURCES = \
 	t/version/version.c
 t_version_version_LDADD = \
-- 
2.25.1

